locales/en/breaches.ftl

```diff
@@ -28,6 +28,11 @@ column-company = COMPANY
 column-breached-data = BREACHED DATA
 column-detected = DETECTED
 
+# "Resolved" is shown next to a breach if all recommended actions in response to the breach have been taken.
```

Comment:
Nit (non-blocking): Since this is only a comment this does not matter for localization, but let’s try to stick with “curly quotes” for strings.

```suggestion
# “Resolved” is shown next to a breach if all recommended actions in response to the breach have been taken.
```

Reply:
Heh, I gotta find out how to type them myself - for regular strings I've just copied them from the Figma's :sweat_smile: 

Fixed in 3ba52594.

Reply:
:)